src/Engine.php

```diff
@@ -48,7 +48,7 @@ public function generate( string $input, int $type, bool $comments ) : string {
 					}
 
 					$output .= $this->generator->generateRewrite($explodedLine[0], $explodedLine[1], $type);
-					$output .= "\n\n";
+					$output .= "\n";
```

Comment:
This just messes up the output formatting…

Reply:
Just to get a shorter ouput...